src/Tickets/Commerce/Gateways/PayPal/Gateway.php

```diff
@@ -128,4 +128,21 @@ public function get_logo_url() {
 	public function get_subtitle() {
 		return __( 'Enable payments through PayPal, Venmo, and credit card', 'event-tickets' );
 	}
+	
+	/**
+	 * @inheritDoc
+	 */
+	public static function is_enabled() {
+		if ( ! static::should_show() ) {
+			return false;
+		}
+		
+		$option_value = tribe_get_option( static::get_enabled_option_key() );
+		if ( '' !== $option_value ) {
```

Comment:
this is a boolean setting, so if it's not set, shouldn't we return false instead of checking for a connection?

Reply:
The purpose of this is to serve those that were already using PayPal in Tickets Commerce prior to having an `enabled` flag to set, so we're setting the default to rely on the connection status. When they upgrade to the newest version of ET, we don't want their sales to be disrupted.

Reply:
If the `enabled` flag IS set, it returns either a 1 or 0. That's why I'm looking explicitly for ''